prometheus/src/main/scala/filodb/prometheus/parse/AntlrParser.scala

```diff
@@ -103,9 +103,11 @@ class AntlrParser extends PromQLBaseVisitor[Object] {
 
   override def visitSubquery(ctx: PromQLParser.SubqueryContext): SubqueryClause = {
     val list = ctx.DURATION
-    val window = parseDuration(list.get(0))
-    val step = if (list.size() > 1) Some(parseDuration(list.get(1))) else None
-    SubqueryClause(window, step)
+    val windowMs = parseDuration(list.get(0))
```

Comment:
is it possible the list is empty?

Reply:
Nope, since it wouldn't otherwise be parsed as a `DURATION`: https://github.com/filodb/FiloDB/blob/7ba04ffc47385d165b9733393f5787dcedbd9274/prometheus/src/main/java/filodb/prometheus/antlr/PromQL.g4#L184